### PR TITLE
added wallet logo and name when is connected with WC

### PR DESCRIPTION
**Pull request by chescalante**
Allows the user to view which wallet is using when connected with wallet connect.

Screen Shot 2021-09-10 at 16 01 30

We had to rely on the peer name setted by each wallet (see warning in the comments).

### CODE REVIEW

File: src/Core.tsx [hunk 1]
```diff
@@ -403,6 +403,8 @@ export class Core extends React.Component<IModalProps, IModalState> {
       this.setState(INITIAL_STATE)
     }
 
+    console.log('selectedProviderUserOption', provider, provider?.wc?._clientMeta, provider?.wc)
```
Comment (jessgusclark) [inline src/Core.tsx:406]: We should remove this console log statement.

File: src/i18n.ts [hunk 2]
```diff
@@ -34,7 +34,8 @@ const resources = {
       Cancel: 'Cancel',
       'Wallet address': 'Wallet address',
       Network: 'Network',
-      'Don’t show again': 'Don’t show again'
+      'Don’t show again': 'Don’t show again',
```
Comment (jessgusclark) [inline src/i18n.ts:37]: This apostrophe _could_ cause encoding issues with some browsers especially on windows. For the key, you could leave it out, and the value either escape it with \'\ or use the encoded version of `&#39;`

i.e.: `Don&#39;t show again`

File: src/providers/injectedProviders.ts [hunk 3]
```diff
@@ -45,11 +45,15 @@ export const checkRLoginInjectedProviders = (providers: IProviderUserOptions[])
 
   // loop through list above to see if one matches
   injectedProviders.map((item: any) => {
+    console.log('injectedProviders', item)
```
Comment (jessgusclark) [inline src/providers/injectedProviders.ts:48]: console log and below.

**Review by jessgusclark — CHANGES_REQUESTED** [review]
Solid work! Just some nit picky stuff.

**Review by ilanolkies — APPROVED** [review]
All good here! Ledger working fine too

**Pull request merged**